modules/luci-mod-status/htdocs/luci-static/resources/view/status/channel_analysis.js

```diff
@@ -342,6 +342,29 @@ return view.extend({
 					}
 				}
 
+				if (res.he_operation != null && res.he_operation.channel_width > 20) {
```

Comment:
perhaps just use optional chaining to avoid the null checks:
```js
if (res.he_operation?.channel_width > 20) {
```

Reply:
Done